Jenkinsfile

```diff
@@ -18,9 +18,11 @@ def runTests(String testPath, Boolean separateMakeStep=true) {
 }
 
 def runTestsWin(String testPath) {
-    bat "runTests.py -j${env.PARALLEL} ${testPath} --make-only"
-    try { bat "runTests.py -j${env.PARALLEL} ${testPath}" }
-    finally { junit 'test/**/*.xml' }
+    withEnv(['PATH+TBB=./lib/tbb']) {
```

Comment:
Will this be built in `stan/lib/tbb`? If so, that is fine, just double checking as you set the path in `cmdstan `to `cmdstan\stan\lib\stan_math\lib\tbb`. After this is question is resolved I will approve.

Reply:
argh... thanks for catching. This is wrong (copy & paste error)... will fix. Thx